packages/autocannon-ui-frontend/src/main.jsx

```diff
@@ -2,11 +2,6 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 
 import './index.css'
-import App from './components/App'
+import App from './components/App.jsx'
```

Comment:
any reason to append the file extension?

Reply:
My eslint IDE plugin was complaining about unresolved dep, which now I realised that it was missing configuration option for resolver extensions... Added that and changed all the imports to be without `jsx` for simplicity sake.